src/gazebo_ros_pulse_lidar.cpp

```diff
@@ -1,14 +1,28 @@
+/*
+ * Copyright 2020 Naval Postgraduate School 
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
 #ifndef _GAZEBO_ROS_PULSE_LIDAR_HH_
 #define _GAZEBO_ROS_PULSE_LIDAR_HH_
```

Comment:
Remove lines 18, 19, and 247.  These preprocessor directives are for header files (.h), not source body files (.cpp).  They are important for headers.  If header definitions are included twice, the compilation will fail, and these prevent that.  They have no meaning for source files which are loaded exactly once.